src/hammer-vlsi/test.py

```diff
@@ -194,6 +194,12 @@ def step(self) -> bool:
         """
         pass
 
+    def tool_config_prefix(self):
+        return "empty"
+
+    def version_number(self, version:str):
+        return 1
+
```

Comment:
Maybe all the test tools should mix in this instead of copying over and over again?